aiidalab_launch/instance.py

```diff
@@ -65,6 +65,7 @@ class AiidaLabInstanceStatus(Enum):
     profile: Profile
     _image: docker.models.images.Image = None
     _container: Container = None
+    _protocol: str = "http"
```

Comment:
I'm wondering whether the default value should be `None` instead. Basically we do not know what the protocol is unless the container was started. In that case we should reset this back to `None` when we stop the container. What do you think?

Reply:
Hmm, not sure this is worth the extra complexity, then we would need to special-case the url() function.

Reply:
Ok, fair enough.